modules/metering-install-operator.adoc

```diff
@@ -8,11 +8,41 @@
 
 .Procedure
 
+
+
+. Create a Namespace for the Metering Operator:
+
+.. Create a Namespace object YAML file (for example, `metering-namespace.yaml`) for the Metering Operator:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-metering <1>
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
+    openshift.io/cluster-monitoring: "true"
```

Comment:
We may want to call out the purpose of the `openshift.io/node-selector` annotation and the `openshift.io/cluster-monitoring` label in separate bullet points. Or we could have a single bullet point that says to just mirror everything listed in this namespace manifest like cluster-logging does in their docset.